assets/app/components/site/editor/prosemirror.js

```diff
@@ -0,0 +1,55 @@
+
+import React from 'react';
+import prosemirror from 'prosemirror';
+import { exampleSetup, buildMenuItems } from 'prosemirror/dist/example-setup'
+import { defaultMarkdownParser, defaultMarkdownSerializer } from 'prosemirror/dist/markdown';
+import { menuBar } from 'prosemirror/dist/menu';
+import { schema } from 'prosemirror/dist/schema-basic';
+
+const menu = buildMenuItems(schema);
+
+class Prosemirror extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toMarkdown = this.toMarkdown.bind(this);
+  }
+
+  componentDidMount() {
+    this.editor = new prosemirror.ProseMirror({
+      doc: defaultMarkdownParser.parse(this.props.initialMarkdownContent),
+      place: document.querySelector('#js-prosemirror-target'),
```

Comment:
Not a huge deal since we are only calling this one, but in general it is always better to use getElementById when you can